performance_regression_test.py

```diff
@@ -797,3 +798,109 @@ def build_histogram(self, workload: PerformanceTestWorkload, test_type: Performa
 
         self.update_hdrhistograms(histogram_name='test_histogram_by_interval',
                                   histogram_data=histogram_data_by_interval)
+
+
+class PerformanceRegressionUpgradeTest(PerformanceRegressionTest, UpgradeTest):  # pylint: disable=too-many-ancestors
+    def __int__(self, *args):
+        super().__init__(*args)
+        self.cluster = None  # pylint: disable=attribute-defined-outside-init
+        self.tester = None  # pylint: disable=attribute-defined-outside-init
+        self.monitoring_set = None  # pylint: disable=attribute-defined-outside-init
```

Comment:
Does still pylint disable comments required?

Reply:
i believe so.. i can try to remove them locally, and run pre-commit, but they are defined outside of `__init__()` method, so the pylint disable should be still required

Reply:
Actually you have typo here. The letter `i` is lost in the `__int__` method name.
So, lines 806-808 make no any effect to the logic.
If the code did work for you, then the definition of these attributes was not really needed...